src/Microsoft.Health.Dicom.Core/Features/Store/Entries/DicomInstanceEntryReaderManager.cs

```diff
@@ -26,8 +26,6 @@ public DicomInstanceEntryReaderManager(IEnumerable<IDicomInstanceEntryReader> di
     /// <inheritdoc />
     public IDicomInstanceEntryReader FindReader(string contentType)
     {
-        EnsureArg.IsNotNull(contentType, nameof(contentType));
```

Comment:
Is a `ContentType` required by the spec or is there some default value we can assume? If it's the latter, then we should populate the default.

Reply:
ContentType is required by spec. No default is specified for this. In addition to the standard, we have also provided a list of accepted ContentType values in the conformance statement.